Decisions.TruBot/Data/TruBotRecordedBot.cs

```diff
@@ -0,0 +1,81 @@
+using DecisionsFramework.Data.ORMapper;
+using DecisionsFramework.Design.Properties;
+using DecisionsFramework.ServiceLayer;
+using DecisionsFramework.Utilities;
+
+namespace Decisions.TruBot.Data
+{
+    [ORMEntity("trubot_recorded_bots")]
+    public class TruBotRecordedBot : AbstractFolderEntity
+    {
+        public TruBotRecordedBot() {}
+
+        [ORMPrimaryKeyField]
+        [PropertyHidden]
+        public string Id { get; set; }
+        
+        [ORMField]
+        public int BotId { get; set; }
+        
+        [ORMField]
+        public string BotName { get; set; }
+        
+        [ORMField]
+        public DateTime InitializedOn { get; set; }
+        
+        [ORMField]
+        public DateTime LastRunOn { get; set; }
+        
+
+        public override void BeforeSave()
+        {
+            if (string.IsNullOrEmpty(Id))
+            {
+                Id = IDUtility.GetNewIdString();
+            }
+            base.BeforeSave();
+        }
+
+        static ORM<TruBotRecordedBot> orm = new();
+
+        public TruBotRecordedBot(int botId)
+        {
+            TruBotRecordedBot bot = GetTruBotRecordByBotId(botId);
+
+            if (bot != null)
+            {
+                orm.Fetch(typeof(TruBotRecordedBot));
```

Comment:
What is this doing? 
When you initialize a new RecordedBot, you fetch them all and don't do anything with the result.